frontend/src/components/Sidebar/RequirementSection.tsx

```diff
@@ -94,10 +94,12 @@ export class RequirementSection extends React.Component<
   }
 
   parseRequirements(reqs: Requirement[]) {
-    return reqs.map(r => this.renderRequirement(r));
+    return reqs.map((r: Requirement, index: number) => (
+      <div key={index + 10000}>{this.renderRequirement(r, index)}</div>
```

Comment:
Why plus 10000?

Reply:
This is old, it was from when I was messing around with the keys. Good catch!!